### DISCUSSION TITLE
Support for `prettier --check`

### DISCUSSION THREAD

**Discussion by Conaclos** [rome/tools#3763] [Q&A] [answered]
Hi!

Is there an equivalent for [`prettier --check`](https://prettier.io/docs/en/cli.html#--check)?

I am using this command in my test script for ensuring that all files were formatted.

Rome could add a flag to the `check` command: `rome check --formatted`

Thanks!

**Comment by MichaReiser** [comment, marked as answer]
You can use `rome ci`, which checks formatting and linting (if not disabled in the configuration). 

```
rome ci <paths> 
```

to check the paths for formatting and linting issues or 

```
rome ci --linter-enabled=false <paths>
```

To only check the formatting.
